docs/src/main/docbook/getting-started.xml

```diff
@@ -428,42 +428,45 @@ Tests run: 0, Failures: 0, Errors: 0, Skipped: 0
+    [INFO] Copying jersey-hk2-3.0.0-RC2.jar to .../simple-heroku-webapp/target/dependency/jersey-hk2-3.0.0-RC2.jar
+    [INFO] Copying jersey-media-jaxb-3.0.0-RC2.jar to .../simple-heroku-webapp/target/dependency/jersey-media-jaxb-3.0.0-RC2.jar
+    [INFO] Copying hk2-utils-3.0.0-M2.jar to .../simple-heroku-webapp/target/dependency/hk2-utils-3.0.0-M2.jar
+    [INFO] Copying jersey-common-3.0.0-RC2.jar to .../simple-heroku-webapp/target/dependency/jersey-common-3.0.0-RC2.jar
+    [INFO] Copying jersey-container-servlet-3.0.0-RC2.jar to .../simple-heroku-webapp/target/dependency/jersey-container-servlet-3.0.0-RC2.jar
+    [INFO] Copying jakarta.annotation-api-2.0.0.jar to .../simple-heroku-webapp/target/dependency/jakarta.annotation-api-2.0.0.jar
     [INFO] Copying jetty-webapp-9.0.6.v20130930.jar to .../simple-heroku-webapp/target/dependency/jetty-webapp-9.0.6.v20130930.jar
-    [INFO] Copying jersey-container-servlet-2.5.jar to .../simple-heroku-webapp/target/dependency/jersey-container-servlet-2.5.jar
-    [INFO] Copying cglib-2.2.0-b21.jar to .../simple-heroku-webapp/target/dependency/cglib-2.2.0-b21.jar
-    [INFO] Copying osgi-resource-locator-1.0.1.jar to .../simple-heroku-webapp/target/dependency/osgi-resource-locator-1.0.1.jar
-    [INFO] Copying hk2-utils-2.2.0-b21.jar to .../simple-heroku-webapp/target/dependency/hk2-utils-2.2.0-b21.jar
-    [INFO] Copying hk2-api-2.2.0-b21.jar to .../simple-heroku-webapp/target/dependency/hk2-api-2.2.0-b21.jar
+    [INFO] Copying jakarta.activation-2.0.0.jar to .../simple-heroku-webapp/target/dependency/jakarta.activation-2.0.0.jar
+    [INFO] Copying osgi-resource-locator-1.0.3.jar to .../simple-heroku-webapp/target/dependency/osgi-resource-locator-1.0.3.jar
+    [INFO] Copying jetty-servlet-9.0.6.v20130930.jar to .../simple-heroku-webapp/target/dependency/jetty-servlet-9.0.6.v20130930.jar
+    [INFO] Copying jetty-xml-9.0.6.v20130930.jar to .../simple-heroku-webapp/target/dependency/jetty-xml-9.0.6.v20130930.jar
+    [INFO] Copying jersey-container-servlet-core-3.0.0-RC2.jar to .../simple-heroku-webapp/target/dependency/jersey-container-servlet-core-3.0.0-RC2.jar
     [INFO] Copying jetty-io-9.0.6.v20130930.jar to .../simple-heroku-webapp/target/dependency/jetty-io-9.0.6.v20130930.jar
+    [INFO] Copying jakarta.ws.rs-api-3.0.0.jar to .../simple-heroku-webapp/target/dependency/jakarta.ws.rs-api-3.0.0.jar
+    [INFO] Copying jakarta.activation-api-2.0.0.jar to .../simple-heroku-webapp/target/dependency/jakarta.activation-api-2.0.0.jar
+    [INFO] Copying jakarta.xml.bind-api-3.0.0.jar to .../simple-heroku-webapp/target/dependency/jakarta.xml.bind-api-3.0.0.jar
+    [INFO] Copying jetty-security-9.0.6.v20130930.jar to .../simple-heroku-webapp/target/dependency/jetty-security-9.0.6.v20130930.jar
     [INFO] Copying jetty-server-9.0.6.v20130930.jar to .../simple-heroku-webapp/target/dependency/jetty-server-9.0.6.v20130930.jar
```

Comment:
TODO: Update Heroku archetype with Jetty 10

Reply:
will be done in a separate PR